charts/validator/templates/deployment.yaml

```diff
@@ -33,12 +33,28 @@ spec:
           hostNetwork: true
           {{- end }}
           image: "{{ .Values.ghost.image.repository }}:{{ .Values.ghost.image.tag | default .Chart.AppVersion }}"
+          {{- if .Values.ghost.commandOverride }}
```

Comment:
@teghnet this makes the `commandOverride` optional, so if its left out, the default container entry point is used